pages/api/[...mockApiRoute].ts

```diff
@@ -1,4 +1,4 @@
-import { addMilliseconds, getDate } from 'date-fns';
+import { addMilliseconds } from 'date-fns';
```

Comment:
good choice not to use `moment` lol

Reply:
Ikr, unfortunately this library has its own issues and I'm contemplating removing it (this took me a while to figure out): https://github.com/date-fns/date-fns/issues/3129